_includes/markdown/CSS.md

```diff
@@ -337,45 +294,11 @@ Avoid mixing min-width and max-width media queries.
 
 ### Breakpoints
 
-Working with Sass, we can take advantages of mixins and avoid having a ton of different breakpoints. Example of a basic media query mixin with a function to check if the breakpoint we want to use has been defined before:
-
-```scss
-/// Register devices widths
-$devices: (
-	mobile-landscape: 480px,
-	tablet: 768px,
-	tablet-landscape: 1024px,
-	laptop: 1280px,
-	desktop: 1440px
-) !default;
-
-/// Verify that the breakpoint width is listed
-///
-/// @param {string} $breakpoint - breakpoint name
-/// @group mediaqueries
-@function get-breakpoint-width($breakpoint) {
-	@if map-has-key($devices, $breakpoint) {
-		@return map-get($devices, $breakpoint);
-	} @else {
-		@warn "Breakpoint #{$breakpoint} wasn't found in $devices.";
-	}
-}
-
-/// Min-width media query
-///
-/// @param {string} $breakpoint - breakpoint name
-/// @group mediaqueries
-@mixin at-least($breakpoint) {
-	$device-width: get-breakpoint-width($breakpoint);
-	@media screen and (min-width: $device-width) {
-		@content;
-	}
-}
-```
+Working with build tools that utilize Sass or PostCSS processing, we can take advantages of reusability and avoid having an unmaintainable number of breakpoints. Using variables and reusable code blocks we can lighten the CSS load and ease maintainability.
```

Comment:
I feel like this is a bit light considering all that was removed. Thoughts?

Reply:
While the previous example _was_ kinda large to take in, could there be basic examples of a small-screen-first styles using either **Sass Variables** or **CSS Custom Media Queries**?

```scss
$m-screen: "min-width: 768px";
$l-screen: "min-width: 1024px";

.some_element {
  padding: 10px;

  @media ($m-screen) {
    padding: 20px;
  }

  @media ($l-screen) {
    padding: 30px;
  }
}
```

```css
@custom-media --m-screen (max-width: 30em);
@custom-media --l-screen (max-width: 30em);

.some_element {
  padding: 10px;

  @media (--m-screen) {
    padding: 20px;
  }

  @media (--l-screen) {
    padding: 30px;
  }
}
```

Feel free to disregard. I love the overall changes! 😄

Reply:
I wouldn't mind adding a small example here. I would either use pseudocode or have an example for Sass and an example for post-CSS. I'd like to stay away from language that tried to define the tool if at all possible.

Reply:
I feel like a few CSS Custom Media queries makes sense as it's part of the new CSS spec (right?). I'm also fine with pseudocode. Your call, Tim.